packages/modelence/src/cron/jobs.ts

```diff
@@ -123,14 +85,18 @@ export async function startCronJobs() {
   }
 }
 
-function sleep(ms: number) {
-  return new Promise((resolve) => setTimeout(resolve, ms));
-}
-
 async function tickCronJobs() {
-  // TODO: periodically check if the locks are still there
-
   const now = Date.now();
+
+  const ownsLock = await acquireLock('cron', {
```

Comment:
The failed lock is saved for 30 seconds, so it will not do another DB request before that.

Reply:
Great, I like the caching logic - so the code that's using this doesn't have to worry about delays / frequency at all and can simply call it and let `acquireLock` take care of caching.